spec/user_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before(:each) do
+    @user = User.create(name: 'Bassi', photo: 'anas.png', bio: 'Student from Nigeria.', posts_counter: 0)
+  end
+
+  context 'with invalid values' do
+    it 'title should be present' do
+      @user.name = nil
+      expect(@user).to_not be_valid
+    end
+
+    it 'PostCounter must be an integer greater than or equal to zero' do
+      @user.posts_counter = nil
+      expect(@user).to_not be_valid
+    end
+  end
+end
```

Comment:
- [ ] Kindly fix the issue for the test. I am getting an error when I run tests. :+1: 

![testerror](https://user-images.githubusercontent.com/70483835/200109331-52fed5e4-29a0-4c8c-ab5d-e398c9af7997.png)